migrations/Version20150924204612.php

```diff
@@ -14,7 +14,7 @@ class Version20150924204612 extends AbstractMigration
     public function up(Schema $schema)
     {
         $todo = $schema->getTable(Table::TODO);
-        $todo->addColumn('deadline', 'datetime', ['default' => null, 'notnull' => false]);
+        $todo->addColumn('deadline', 'string', ['default' => null, 'notnull' => false, 'length' => 30]);
     }
 
```

Comment:
That was the reason why I named Postgres. My idea was to rather save the dates in UTC and handle the conversion on application side, but you are right, `DateTime` handles TZs really well and if we don't need the `datetime` type in the DB, you might as well save it as string in `ISO8601` format.